src/mcp/tools/core/read_resources.ts

```diff
@@ -24,6 +25,7 @@ export const read_resources = tool(
   },
   async ({ uris }, ctx) => {
     if (!uris?.length) {
+      await trackGA4("mcp_read_resource", { resource_name: "listAvaiable" });
```

Comment:
```suggestion
      await trackGA4("mcp_read_resource", { resource_name: "__list__" });
```

would prefer something more like this sentinel i think, and track the same thing in the MCP ListResources method.